### DISCUSSION TITLE
TypeError: setTimeout(...).catch is not a function

### DISCUSSION THREAD

**Discussion by ExistedYear** [LunarTaku/djs-ticket-system#12] [Error]
the ticket system worked perfectly until in created a ticket and when i tried to close it, it didnt work
Error: https://sourceb.in/DJQhxY8K7P
Code: https://sourceb.in/6WcSlxr9EB

**Comment by LunarTaku** [comment]
try and add `await` before the `setTimeout`

**Reply by ExistedYear** [reply]
let me try real quick

**Reply by LunarTaku** [reply]
alright

**Reply by ExistedYear** [reply]
nope. same error.

**Reply by LunarTaku** [reply]
try removing it

**Reply by ExistedYear** [reply]
remove the `await`?

**Reply by LunarTaku** [reply]
no the setTimeout, its useless for the bot its there to delay the delete.

**Reply by ExistedYear** [reply]
oh

**Reply by ExistedYear** [reply]
now its just stuck at saying "deleting ticket" and doesnt delete it and there is no error in console

**Reply by LunarTaku** [reply]
Yeah just remove the timeout not the delete channel lmao
